test/examples/ucx_hello_world.h

```diff
@@ -35,6 +48,72 @@
     } while (0)
 
 
+#define CUDA_FUNC(_func)                                   \
+    do {                                                   \
+        cudaError_t _result = (_func);                     \
+        if (cudaSuccess != _result) {                      \
+            fprintf(stderr, "%s failed: %s",               \
+                    #_func, cudaGetErrorString(_result));  \
+        }                                                  \
+    } while(0)
+
+
+struct mem_type_allocator {
+    void*   (*malloc)(size_t length);
+    void    (*free)(void *address);
+    void*   (*memcpy)(void *dst, const void *src, size_t count);
+    void*   (*memset)(void *dst, int value, size_t count);            
+};
+
+static ucs_memory_type_t test_mem_type = UCS_MEMORY_TYPE_HOST;
+
+#ifdef UCX_HELLO_WORLD_CUDA
+void *cuda_malloc(size_t length)
+{
+    void *ptr;
+    CUDA_FUNC(cudaMalloc(&ptr, length));
+    return ptr;
+}
+
+void *cuda_malloc_managed(size_t length)
+{
+    void *ptr;
+    CUDA_FUNC(cudaMallocManaged(&ptr, length, cudaMemAttachGlobal));
+    return ptr;
+}
+
+void cuda_free(void *address)
+{
+    cudaFree(address);
+}
+
+void *cuda_memcpy(void *dst, const void *src, size_t count)
+{
+    cudaMemcpy(dst, src, count, cudaMemcpyDefault);
+    return dst;
+}
+
+void *cuda_memset(void *dst, int value, size_t count)
+{
+    cudaMemset(dst, value, count);
+    return dst;
+}
+#endif
+
+static struct mem_type_allocator mem_type_allocators[UCS_MEMORY_TYPE_LAST] = {
+    {
+        malloc, free, memcpy, memset
+    },
```

Comment:
you may move `,` to the line below (may help to avoid some paranoidal compilers warning)